src/main/msp/msp_serial.h

```diff
@@ -39,23 +38,24 @@ typedef enum {
 } mspEvaluateNonMspData_e;
 
 #define MSP_PORT_INBUF_SIZE 64
```

Comment:
Large arrays should be transferred in multiple packets (per array item). I'am working on it, but time is scarce ..